Documentation/running-exporters.md

```diff
@@ -0,0 +1,66 @@
+# Running Exporters
+Running exporters and scraping them with Prometheus configured by the prometheus-operator.
+
+## Two approaches to utilizing ServiceMonitors
+What approach is the "best" is currently mostly a "personal" preference.
```

Comment:
Remove line 4 and 5 and combine the other sections into one. And describe, that the goal is always to have the least amount of `ServiceMonitor`s discovering a large number of `Service`s. In certain cases it won't be avoidable to have a 1-1 relationship, but it's not really a preference thing as every `ServiceMonitor` results in Prometheus actually reloading the configuration, which means sample ingestion stops for that time. So it's best to have the least amount of `ServiceMonitor`s, or at least a set that does not change (add, remove, modify) very often.